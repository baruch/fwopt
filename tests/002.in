-A INPUT -i eth0 --src 1.2.3.4 -j ACCEPT
-A INPUT -i eth0 --src 1.2.3.5 -j ACCEPT
