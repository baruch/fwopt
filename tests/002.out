iptables -N chain_1
iptables -A chain_1 --src 1.2.3.5 -j ACCEPT
iptables -A chain_1 --src 1.2.3.4 -j ACCEPT
iptables -A INPUT -i eth0 -j chain_1
